src/pages/auth/verify-login.tsx

```diff
@@ -0,0 +1,93 @@
+import { FormattedMessage, useIntl } from 'react-intl';
+import { useMemo } from 'react';
+import { useRouter } from 'next/router';
+import { toast } from 'react-toastify';
+import { z } from 'zod';
+
+import { DarkButton } from '../../components/decoration/buttons';
+import { FirstSubSection } from '../../components/decoration/textBlocks';
+import SquareField from '../../components/decoration/squares';
+import useOnce from '../../hooks/useOnce';
+
+const VerifyLogin: React.FC = () => {
+  const intl = useIntl();
+  const router = useRouter();
+
+  useOnce(
```

Comment:
@Headscracher I've left `useOnce` here for  because it looks like `router.push` can only be used when `router.isReady` is true which is tested here via the `enabled` option of this hook. There's probably another way to make it work inside `useMemo` though, just let me know (or feel free to adjust it in this branch if you prefer).